src/mach/imports.rs

```diff
@@ -195,9 +216,9 @@ impl<'a> BindInterpreter<'a> {
 	            // throwBadBindingAddress(address, segmentEndAddress, segmentIndex, start, end, p);
 	            // (this->*handler)(context, address, type, symbolName, symboFlags, addend, libraryOrdinal, "", &last);
 	            // address += sizeof(intptr_t);
+                    imports.push(Import::new(&bind_info, libs, segments, start_of_sequence));
                     let seg_offset = bind_info.seg_offset.wrapping_add(ctx.size() as u64);
                     bind_info.seg_offset = seg_offset;
-                    imports.push(Import::new(&bind_info, libs, segments));
```

Comment:
Oh! As for how this could go unnoticed, consider a typical opcode sequence:

```0x0178 BIND_OPCODE_DO_BIND()
0x0179 BIND_OPCODE_SET_SYMBOL_TRAILING_FLAGS_IMM(0x00, __ZNKSt3__119__shared_weak_count13__get_deleterERKSt9type_info)
0x01B9 BIND_OPCODE_ADD_ADDR_ULEB(0xFFDB7F50)
0x01C4 BIND_OPCODE_DO_BIND_ADD_ADDR_ULEB(0x00168778)
0x01C8 BIND_OPCODE_DO_BIND_ADD_ADDR_ULEB(0x00000090)
0x01CB BIND_OPCODE_DO_BIND_ADD_ADDR_ULEB(0x0009A908)
0x01CF BIND_OPCODE_DO_BIND_ADD_ADDR_ULEB(0x000000E0)
0x01D2 BIND_OPCODE_DO_BIND_ADD_ADDR_ULEB(0x000000B8)
0x01D5 BIND_OPCODE_DO_BIND_ADD_ADDR_IMM_SCALED(0x00000060)
0x01D6 BIND_OPCODE_DO_BIND_ADD_ADDR_ULEB(0x000299E8)
0x01DA BIND_OPCODE_DO_BIND_ADD_ADDR_ULEB(0x000277A8)
0x01DE BIND_OPCODE_DO_BIND_ADD_ADDR_ULEB(0x00000BC0)
0x01E1 BIND_OPCODE_DO_BIND_ADD_ADDR_ULEB(0x00000E00)
0x01E4 BIND_OPCODE_DO_BIND_ULEB_TIMES_SKIPPING_ULEB(3, 0x00000030)
```

Bind sequences typically look like "set up stuff, bind, bind, bind, bind, bind", where each bind operation binds and moves the cursor. Binding at the start versus binding at the end is basically an off-by-one error -- the first bind gets missed and an errant bind gets added at the end, but most of the binds in the middle get covered correctly.